packages/react-renderer-demo/src/app/pages/renderer/component-api.md

```diff
@@ -184,8 +184,8 @@ Wizard step <br/>
 |Prop|Type|Description|
```

Comment:
There are some legacy comments about form groups `/renderer/component-api#formgroupwrappedcomponents`. I would probably remove this. I would probably change this to something like supported mappers prop names or something. To let people know that if they want their mappers compatible with the ones living in data-driven forms, they should use this notation.

Reply:
It's actually there `Each mapper provided in `react-forms` provides a default API for standard components.` I will expand it